eessi_bot_job_manager.py

```diff
@@ -85,6 +87,10 @@ def __init__(self):
         cfg = config.read_config()
         job_manager_cfg = cfg[config.SECTION_JOB_MANAGER]
         self.logfile = job_manager_cfg.get(config.JOB_MANAGER_SETTING_LOG_PATH)
+        buildenv_cfg = cfg[config.SECTION_BUILDENV]
+        self.job_name = buildenv_cfg.get(config.BUILDENV_SETTING_JOB_NAME)
+        if len(self.job_name) < 3:
```

Comment:
What if `job_name` is not set, this will go down pretty hard?

```python
>>> len(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```

So:

```suggestion
        if self.job_name and len(self.job_name) < 3:
```